cpp/src/gandiva/function_registry_string.cc

```diff
@@ -439,6 +439,14 @@ std::vector<NativeFunction> GetStringFunctionRegistry() {
                      kResultNullIfNull, "right_utf8_int32",
                      NativeFunction::kNeedsContext),
 
+      NativeFunction("binary", {}, DataTypeVector{binary()}, binary(), kResultNullIfNull,
+                     "castBINARY_binary",
+                     NativeFunction::kNeedsContext | NativeFunction::kCanReturnErrors),
```

Comment:
It doesn't requried context and canreturnerrors flags.

Reply:
Ok, these two flags was removed.